capstone/src/main/webapp/group.html

```diff
@@ -3,13 +3,24 @@
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <link
-      href="http://fonts.googleapis.com/css?family=Roboto"
-      rel="stylesheet"
-      type="text/css"
-    />
+    <link rel="stylesheet" href="styles/group.css"></link>
     <meta http-equiv="X-UA-Compatible" content="ie=edge" />
     <title>Group</title>
   </head>
-  <body></body>
+
+  <body>
+    
+    <div class="group-content">
+
+      <div class="group-header">
+      </div>
```

Comment:
nit: Make this one line since it doesn't have any children
```suggestion
      <div class="group-header"></div>
```